syndiffix/bucket.py

```diff
@@ -240,11 +240,10 @@ def _harvest_node(node: Node, harvested_nodes: BucketsCache, unsafe_rng: Random)
     return harvested_nodes[node]
 
 
-def harvest(node: Node) -> Buckets:
+# The passed RNG isn't safe for anonymization, so it can only be applied to already anonymized data.
+# Used for randomly matching subintervals during refining.
+def harvest(node: Node, unsafe_rng: Random = Random(0)) -> Buckets:
```

Comment:
```py
unsafe_rng: Random = Random(0)
```

The above is non-sticky because Python initializes default values once globally. We probably don't care if it's for tests only.